crates/fmt/testdata/ReturnStatement/fmt.sol

```diff
@@ -23,7 +23,9 @@ contract ReturnStatement {
                 0x00;
         }
 
-        if (val == 1) return 1;
+        if (val == 1) {
+            return 1;
+        }
```

Comment:
this is fine, we keep the block by default because the original uses a block, even if it would fit in one line. This is broken in the current formatter